src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -115,11 +118,14 @@ def _validate_stack_requirements(stack: "Stack") -> Tuple[bool, str]:
             # Validate that the container registry is not local.
             container_registry = stack.container_registry
             if container_registry and container_registry.config.is_local:
-                return False, (
-                    f"The Vertex orchestrator does not support local "
-                    f"container registries. You should replace the component '"
-                    f"{container_registry.name}' "
-                    f"{container_registry.type.value} to a remote one."
+                return (
+                    False,
+                    (
+                        "The Vertex orchestrator does not support local"
```

Comment:
Any reason for this formatting change? Im trying to see if something changed but dont notice it

Reply:
Hey, I don't know! Let me check if I messed up something with `black`